packages/components/src/box-control/all-input-control.tsx

```diff
@@ -25,25 +33,41 @@ export default function AllInputControl( {
 	setSelectedUnits,
 	...props
 }: BoxControlInputControlProps ) {
+	const generatedId = useInstanceId( AllInputControl, 'box-control-input' );
+	const inputId = [ generatedId, LABELS.all ].join( '-' );
```

Comment:
```suggestion
	const inputId = useInstanceId( AllInputControl, 'box-control-input-all' );
```

I think we should just do this because `LABELS.all` will be a translated string and not suitable for `id`s.